source/fuzz/transformation_add_constant_composite.cpp

```diff
@@ -77,14 +77,9 @@ bool TransformationAddConstantComposite::IsApplicable(
       // BufferBlock.  The SPIR-V spec does not explicitly disallow this, but it
       // seems like a strange thing to do, so we disallow it to avoid triggering
       // low priorty edge case issues related to it.
-      for (auto decoration : {SpvDecorationBlock, SpvDecorationBufferBlock}) {
-        if (!ir_context->get_decoration_mgr()->WhileEachDecoration(
-                composite_type_instruction->result_id(), decoration,
-                [](const opt::Instruction & /*unused*/) -> bool {
-                  return false;
-                })) {
-          return false;
-        }
+      if (fuzzerutil::HasBlockOrBufferBlockDecoration(
```

Comment:
It would probably be good to adjust docs in the header as well.

Reply:
Looks like I already had (but I then saw a typo which I have now fixed.)

Reply:
...but let me know if you meant something different.

Reply:
Yes, you are right. I got confused a little.